packages/@sanity/form-builder/src/inputs/SelectInput.js

```diff
@@ -27,6 +28,10 @@ export default class StringSelect extends React.Component<Props> {
     value: ''
   }
 
+  componentDidMount() {
+    this.name = uniqueId('RadioName')
```

Comment:
It would be better if this is set either in the constructor or as a class field.